include/dts-functions.sh

```diff
@@ -774,6 +774,26 @@ board_config() {
       esac
       shopt -u nocasematch
       ;;
+    "HARDKERNEL")
+      HAVE_EC="false"
+      NEED_EC_RESET="false"
+      NEED_SMBIOS_MIGRATION="true"
+      NEED_BLOB_TRANSMISSION="false"
+      PROGRAMMER_BIOS="internal"
+      case "$SYSTEM_MODEL" in
+        "ODROID-H4")
+          DASHARO_REL_NAME="hardkernel_odroid_h4"
+          DASHARO_REL_VER_DPP="0.9.0-rc1"
```

Comment:
```suggestion
          DASHARO_REL_VER_DPP="0.9.0"
```

When the release is out it will not have RC1 suffix. ALso we do not install/deploy RC version via DTS